frontend/src/components/DefenceBox/PromptEnclosureDefenceMechanism.test.tsx

```diff
@@ -155,12 +147,13 @@ describe('PromptEnclosureDefenceMechanism component tests', () => {
 
 	test('when xml tagging radio button is selected, it is toggled on and others are toggled off', async () => {
 		const newMockDefences = [...mockDefences];
+		newMockDefences[0].isActive = false;
 		newMockDefences[1].isActive = true;
 
 		const { user } = renderComponent(newMockDefences);
 		const radioButton = screen.getByRole('radio', {
 			name: 'xml tagging',
-		});
+		}) as Element;
```

Comment:
Why?

Reply:
I had used it in the other tests as think i was getting something like a Element vs HTMLElement warning on the user button click

Reply:
I'll remove it, then we're good to go.